packages/components/src/DataTable/DataTable.test.tsx

```diff
@@ -0,0 +1,185 @@
+import React from "react";
+import { fireEvent, render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { DataTable } from "./DataTable";
+import { columns, data } from "./test-utilities";
+
+describe("when rendering a Basic Table", () => {
+  beforeEach(() => {
+    render(<DataTable data={data} columns={columns} />);
+  });
+
+  it("renders all headers", () => {
+    const tableHeader = screen.getAllByRole("row")[0];
+
+    expect(within(tableHeader).getByText(/Name/i)).toBeInTheDocument();
+    expect(within(tableHeader).getByText(/House/i)).toBeInTheDocument();
+    expect(within(tableHeader).getByText(/Region/i)).toBeInTheDocument();
+    expect(within(tableHeader).getByText(/Sigil/i)).toBeInTheDocument();
+    expect(within(tableHeader).getByText(/Alive/i)).toBeInTheDocument();
+  });
+
+  it("renders all rows", () => {
+    const [, ...bodyRows] = screen.getAllByRole("row");
+
+    expect(bodyRows).toHaveLength(data.length);
+  });
+});
+
+describe("when using pagination", () => {
+  beforeEach(() => {
+    render(
+      <DataTable
+        data={data}
+        columns={columns}
+        pagination={{ manualPagination: false, itemsPerPage: [5, 10, 15] }}
+      />,
+    );
+  });
+
+  it("renders table with pagination info", () => {
+    const paginationInfo = screen.getByText(/Showing 1-10 of 13 items/i);
```

Comment:
if itemsPerPage has 5 as an option, I am surprised it is not the default selection 🤔 ie. "1-5 of 13 items"

Reply:
Oh, that might be a bug. Nice catch.